tests/scenarios/v2-addon-dev-test.ts

```diff
@@ -0,0 +1,227 @@
+import path from 'path';
```

Comment:
Be sure to test this scenario with peer deps: https://github.com/embroider-build/embroider/issues/1179#issuecomment-1090462587